docs/impact_health.rst

```diff
@@ -0,0 +1,62 @@
+#################
+Impact and Health
+#################
+
+The figures on this page give a sense of the level of impact ``astropy`` has on the
+astronomy community and research in the field, as well as the amount of
+development effort contributed over time (the codebase's "health"). A major
+positive trend is that citations to ``astropy``, and by proxy its usage in
+research, has been consistently growing since The Astropy Project was created.
+However, the amount of developer resources has remained small, with a limited
+number of volunteer maintainers met with an increasing amount of development
+responsibility. Hence, we welcome any help that you can give!
+
+First considering the citations, this figure estimates ``astropy``'s usage in
+astronomy research by its yearly publication impact, using citations drawn from
+the NASA ADS database. The continual growth in citations shows the broad and
+increasing usage of ``astropy`` in astronomy and scientific research, as well as
+the high impact that contributions to ``astropy`` can have.
+
+|Citation figure|
+
+To next visualize the size of the developer community contributing to the
+``astropy`` core library, this figure shows the number of people authoring commits
+over time. While each year ``astropy`` is cited more in papers, the amount of
+developers and especially those contributing multiple times is modest and
+largely static.
+
+|Commits figure|
+
+The workload to maintain ``astropy`` can be traced through the number of issues and
+pull requests open and closed in the ``astropy`` core library over time. The
+long-term increase in overall pull requests, along with a comparable amount of
```

Comment:
Hmm interesting that issues and PRs open/close kinda track each other, respectively. What does this mean?

Also, will your conclusion hold if I tell you we have a stale bot to auto close stale PRs? See https://docs.astropy.org/en/latest/development/maintainers/maintainer_workflow.html#stale-policies

Reply:
I assume opens and closes track each other for both issues and PRs because maybe a majority of issues can be closed fairly quickly by new PRs, which themselves don't need to be open long. 

https://github.com/jeffjennings/repo_stats/blob/89408b954d846d5a8e0bee169d08be7915f5b95c/repo_stats/git_metrics.py#L287 has a list of bots whose commits I ignore when computing commit statistics (if you know of others, please add them). Since that list already has `github-actions` and it looks like the stale bot uses this ID (e.g. https://github.com/astropy/astropy/issues/16435), the stale bot commits are excluded from these plots.

Reply:
Might have to rerun after https://github.com/astropy/repo_stats/issues/6 is resolved?

Reply:
That's now closed.